apps/product/tasks.py

```
@@ -1,207 +1,983 @@
+                        elif W_product and (not yandexmarket_product):
+                            product_sale_y = ProductSale.objects.get_or_create(product=W_product,company=company,date=date,warehouse=warehouse,marketplace_type="yandexmarket")
+                    else:
+                        product_obj, created_p = Product.objects.get_or_create(vendor_code=vendor_code, barcode=barcode, marketplace_type="yandexmarket")
+                        if not ProductSale.objects.filter(
+                            product=product_obj,
+                            company=company,
+                            date=date,
+                            warehouse=warehouse,
+                            marketplace_type="yandexmarket"
+                        ):
+                            product_s = ProductSale.objects.get_or_create(
```

Ruff

Local variable `product_s` is assigned to but never used

Remove assignment to unused variable `product_s`

(F841)